specs/marketplace.md

```diff
@@ -185,178 +174,159 @@ struct Content {
 
 ```
 
-Notes about some parameters:
+The meaning of the Request parameters is given below:
+
+> Note that not all parameters are defined. I think that in a formal spec we need to introduce all of them, even shortly.
 
 `cid` 
 
 An identifier used to locate the Manifest representing the dataset.
-- MUST be a [CIDv1](https://github.com/multiformats/cid#cidv1) with sha-256 based [multihash](https://github.com/multiformats/multihash).
-- Data it represents SHOULD be discoverable in the network, otherwise the Request will get canceled. 
+- MUST be a [CIDv1](https://github.com/multiformats/cid#cidv1), SHA-256 [multihash](https://github.com/multiformats/multihash).
+- The data it represents SHOULD be discoverable in the network; otherwise, the Request MUST be canceled.
```

Comment:
I don't think that's a correct use of a semicolon. The "otherwise" coordinating conjunction is used instead of a semicolon. 
From https://www.grammarly.com/blog/semicolon/:
> What is correct semicolon use? The most common semicolon use is joining two [independent clauses](https://www.grammarly.com/blog/independent-and-dependent-clauses/) without using a [coordinating conjunction](https://www.grammarly.com/blog/coordinating-conjunctions/) like and.
>
> Coordinating conjunctions to join independent clauses
> An [independent clause](https://www.grammarly.com/blog/independent-and-dependent-clauses/) is a clause that can stand alone as a simple sentence. When two independent clauses are logically closely related, it often makes sense to combine them into a [compound sentence](https://www.grammarly.com/blog/compound-sentence/) with the help of a coordinating conjunction and a comma.
>
>    _I wanted something to eat, so I looked in the fridge._
> 
>    _Reading fiction is beneficial, for it can make us more empathetic._
>
>    _The host borrowed dozens of folding chairs, yet there were not enough for the guests._
>
>    _We liked the play’s story, but we didn’t care for its staging._

Reply:
Thanks @emizzle for explanation. I will get rid of semi...

Reply:
I will remove semicolons....